sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/jsonExpressions.scala

```diff
@@ -796,3 +796,75 @@ case class SchemaOfJson(
 
   override def prettyName: String = "schema_of_json"
 }
+
+/**
+ * A function that returns the number of elements in outer JSON array.
+ */
+@ExpressionDescription(
+  usage = "_FUNC_(jsonArray) - Returns the number of elements in outer JSON array.",
+  arguments = """
+    Arguments:
+      * jsonArray - A JSON array. An exception is thrown if any other valid JSON strings are passed.
+          `NULL` is returned in case of an invalid JSON.
```

Comment:
Shall we mention `NULL` input explicitly?
```
- `NULL` is returned in case of an invalid JSON.
+ `NULL` is returned in case of `NULL` or an invalid JSON
```